### PR TITLE
doc: Grammarly pass on all pages

### PR DESCRIPTION
**Pull request by charlypoly**

### CODE REVIEW

File: website/docs/custom-codegen/plugin-structure.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ title: Write your first Plugin
 
 To get started with writing your GraphQL Code-Generator plugin, start by creating a simple `my-plugin.js` file in your project, with the following content:
 
-```js
+"`js
```
Comment (B2o5T) [inline website/docs/custom-codegen/plugin-structure.md:10]: Did prettier break it?

File: website/docs/custom-codegen/plugin-structure.md [hunk 2]
```diff
@@ -73,9 +75,9 @@ module.exports = {
 
 ## Add plugin configuration
 
-The third argument of your function is `config` and it includes an object with the configuration passed to your plugin.
+The third argument of your function is `config,` and it includes an object with the configuration passed to your Plugin.
```
Comment (B2o5T) [inline website/docs/custom-codegen/plugin-structure.md:78]:
```suggestion
The third argument of your function is `config`, and it includes an object with the configuration passed to your Plugin.
```

**Comment by B2o5T** [comment]
It’s so good 💪🏻, you used Grammarly on VSCode? Can we run some checks on CI?

File: website/docs/config-reference/codegen-config.md [hunk 3]
```diff
@@ -75,9 +88,9 @@ Here are the supported options that you can define in the config file (see [sour
 
 - **`pluginLoader`** - If you are using the programmatic API in a browser environment, you can override this configuration to load your plugins in a way different than `require`.
 
-- **`pluckConfig`** - Allows you to override the configuration for `graphql-tag-pluck`, the tool that extracts your GraphQL operations from your code files.
+- **`pluckConfig`** - Allows you to override the configuration for `graphql-tag-pluck`. This tool extracts your GraphQL operations from your code files.
 
-  - **`pluckConfig.modules`** - An array of `{ name: string, identifier: string }` that will be used to track down your `gql` usages and imports. Use this if your code files imports `gql` from another library, or you have a custom `gql` tag. `identifier` is the named export, so don't provide it if the tag function is imported as default.
+  - **`pluckConfig.modules`** - An array of `{ name: string, identifier: string }` uses to track down your `gql` usages and imports. Use this if your code files import` gql` from another library or have a custom `gql` tag. `identifier` is the named export, so don't provide it if the tag function is imported as default.
```
Comment (B2o5T) [inline website/docs/config-reference/codegen-config.md:93]:
```suggestion
  - **`pluckConfig.modules`** - An array of `{ name: string, identifier: string }` uses to track down your `gql` usages and imports. Use this if your code files import `gql` from another library or have a custom `gql` tag. `identifier` is the named export, so don't provide it if the tag function is imported as default.
```

File: website/docs/config-reference/codegen-config.md [hunk 4]
```diff
@@ -105,17 +125,24 @@ generates:
       - typescript-operations
 ```
 
-You can load an `.env` file by adding the `-r dotenv/config` option to your CLI command.
+You can load a `.env` file by adding the `-r dotenv/config` option to your CLI command.
```
Comment (B2o5T) [inline website/docs/config-reference/codegen-config.md:128]: Did it more correct ?

File: website/docs/config-reference/schema-field.md [hunk 5]
```diff
@@ -156,7 +156,7 @@ schema:
 
 ##### `commentDescriptions`
 
-This will convert all deprecated form of Graphql comments (marked with `#`) into a GraphQL descriptions (marked with `"`) during the parsing phase.
+When enabled, converts all deprecated forms of Graphql comments (marked with `#`) into a GraphQL description (marked with `"`) during the parsing phase.
```
Comment (B2o5T) [inline website/docs/config-reference/schema-field.md:159]:
```suggestion
When enabled, converts all deprecated forms of GraphQL comments (marked with `#`) into a GraphQL description (marked with `"`) during the parsing phase.
```

File: website/docs/custom-codegen/plugin-structure.md [hunk 6]
```diff
@@ -31,9 +31,11 @@ Now, run the `@graphql-codegen/cli` using `graphql-codegen` command, and it will
 
 ## Using the `GraphQLSchema`
 
-The first argument of your `plugin` function is the `GraphQLSchema`. The code-generator will make sure to merge all `GraphQLSchema`s into a single, easy-to-use object, that you can use to create your output.
+The first argument of your `plugin` function is the `GraphQLSchema`.
+
+Then, the code-generator will make sure to merge all `GraphQLSchema`'s into a single, easy-to-use object that you can use to create your output.
 
-You can find the full, typed, [API of `GraphQLSchema` here](https://github.com/graphql/graphql-js/blob/2df59f18dd3f3c415eaba57d744131a674079ddf/src/type/schema.ts#L129)
+You can find the full-typed [API of `GraphQLSchema` here](https://github.com/graphql/graphql-js/blob/2df59f18dd3f3c415eaba57d744131a674079ddf/src/type/schema.ts#L129)
```
Comment (B2o5T) [inline website/docs/custom-codegen/plugin-structure.md:38]:
```suggestion
You can find the fully-typed [API of `GraphQLSchema` here](https://github.com/graphql/graphql-js/blob/2df59f18dd3f3c415eaba57d744131a674079ddf/src/type/schema.ts#L129)
```
I think

File: website/docs/config-reference/codegen-config.md (hunk 4 above)
Comment (charlypoly) [reply website/docs/config-reference/codegen-config.md:128]: Not sure about the grammatical rule here 😅

File: website/docs/custom-codegen/plugin-structure.md (hunk 1 above)
Comment (charlypoly) [reply website/docs/custom-codegen/plugin-structure.md:10]: yes

Comment (charlypoly) [inline website/docs/custom-codegen/plugin-structure.md:10]:
```suggestion
```js
```

**Comment by charlypoly** [comment]
> It’s so good 💪🏻, you used Grammarly on VSCode? Can we run some checks on CI?

@B2o5T I used Grammarly in VSCode with this plugin: https://github.com/znck/grammarly

We could try to do a CI task by using https://www.npmjs.com/package/@stewartmcgown/grammarly-api
and inspiration from: https://github.com/KaamiDev/Grammarly-Discord-Bot

**Review by B2o5T — APPROVED** [review]
LGTM! Thanks

**Pull request merged**